chromium_src/chrome/browser/ui/views/tabs/tab_strip.cc

```diff
@@ -45,8 +51,19 @@
     return tabs::CalculateBoundsForVerticalDraggedViews(views, tab_strip_); \
   }
 
+#define BRAVE_PAINT_CHILDREN                                              \
+  if (base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs) && \
+      tabs::utils::ShouldShowVerticalTabs(tab_strip_->GetBrowser())) {    \
+    for (const ZOrderableTabContainerElement& child : orderable_children) \
+      if (!child.view()->layer()) {                                       \
+        child.view()->Paint(paint_info);                                  \
+      }                                                                   \
+    return;                                                               \
+  }
+
 #include "src/chrome/browser/ui/views/tabs/tab_strip.cc"
 
+#undef BRAVE_PAINT_CHILDREN
```

Comment:
this should include a class name and be `BRAVE_TAB_DRAG_CONTEXT_IMPL_PAINT_CHILDREN`, but other macro are also not named properly, so.. I'd rename them all to be in sync with the code style.

Reply:
Thanks! Renamed macro in this file.

Reply:
don't forget `#undefs`.

Reply:
Shame on me 😹  done.